firefox_media_tests/playback/test_video_playback.py

```diff
@@ -28,12 +28,15 @@ def tearDown(self):
     def test_playback_starts(self):
         with self.marionette.using_context('content'):
             for url in self.test_urls:
+                print url
```

Comment:
The url is already logged by marionette in the `VideoPuppeteer` constructor. Is this left over from debugging?